stories/assets/scss/_breakpoints.scss

```diff
@@ -5,22 +5,30 @@
       @content;
     }
   }
+  @else if $point==mediumfromtab {
+    /* medium design tab */
+    @media (max-width: 64em) {
```

Comment:
@Ashish-srij it should be max-width: 63.9375em as we already mentioned min-width :64rem

Reply:
Yes, I have made a CSS for max-width only.

Reply:
@Ashish-srij please correct the breakpoints which should be used for page templates